tests/typings/webdriverio/types/async.d.ts

```diff
@@ -4,12 +4,6 @@
 
 // module should be "webdriverio" if used within `ts` file instead of `d.ts`
 declare module WebdriverIO {
-    interface BrowserObject {
```

Comment:
We now have proper typings for `MultiremoteBrowserObject`, so this is not necessary any more